frontend/src/e2e-playwright/browser.ts

```diff
@@ -77,10 +78,33 @@ async function takeScreenshots(namePrefix: string): Promise<void> {
   }
 }
 
+function configurePage(page: Page) {
+  page.on(
+    'console',
+    (msg) =>
+      void (async () => {
+        const args: unknown[] = []
+        for (const arg of msg.args()) {
+          const value = (await arg.jsonValue()) as unknown
+          args.push(value)
+          await arg.dispose()
```

Comment:
Is the dispose call needed? It's not used in the example: https://playwright.dev/docs/api/class-page#pageonconsole

Reply:
Hmm, maybe not...I added it because I remembered that JSHandle values retain memory in the browser unless disposed manually.
But it's probably not a big problem since all memory is reclaimed once the page/context/browser is closed